js/chat/chat.scss

```diff
@@ -55,6 +55,7 @@ shiny-chat-container {
   shiny-chat-input {
     margin-top: auto;
     position: sticky;
+    background-color: var(--bs-body-bg, white);
```

Comment:
The intention here is to prevent message content from being visible below the input 

**Before**

<img width="592" alt="Screenshot 2024-07-08 at 10 47 31 AM" src="https://github.com/posit-dev/py-shiny/assets/1365941/fcb6c205-682b-4225-8586-b79816a2b4b5">


**After**

<img width="599" alt="Screenshot 2024-07-08 at 10 49 43 AM" src="https://github.com/posit-dev/py-shiny/assets/1365941/76f6f7bd-9a8e-45ef-81fd-8628933eaa87">